examples/app/multi_node/train_pytorch.py

```diff
@@ -22,7 +22,12 @@ def distributed_train(local_rank: int, main_address: str, main_port: int, num_no
 
     # 2. PREPARE DISTRIBUTED MODEL
     model = torch.nn.Linear(32, 2)
-    device = torch.device(f"cuda:{local_rank}") if torch.cuda.is_available() else torch.device("cpu")
+    if torch.cuda.is_available():
```

Comment:
I would remove MPS from the multi-node scripts here. There is no backend that can support this today, so it's not a realistic use case.